.github/actions/upload/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "filecoin-pin-upload-action-helper",
+  "private": true,
+  "type": "module",
+  "version": "1.0.16",
+  "description": "Helper runner for Filecoin Pin Upload GitHub Action",
+  "dependencies": {
+    "@actions/artifact": "^2.3.2",
+    "@filoz/synapse-sdk": "^0.28.0",
+    "@octokit/rest": "^22.0.0",
+    "ethers": "^6.15.0",
+    "filecoin-pin": "^0.6.0",
```

Comment:
if we plan to keep the action in the `filcoin-pin` repo, not monorepo, we should probably target the local version instead of the published version